src/common/pf_alarm.c

```diff
@@ -915,11 +915,18 @@ static int pf_alarm_apms_a_data_req(
             }
             else if (p_fixed->pdu_type.type == PF_RTA_PDU_TYPE_ERR)
             {
-               pf_put_pnet_status(true, p_pnio_status, PF_FRAME_BUFFER_SIZE, p_buf, &pos);
+               if (p_pnio_status != NULL)
```

Comment:
Getting here with p_pnio_status is an internal p-net error? If so an assert seems more appropriate.